web_src/js/markup/codecopy.ts

```diff
@@ -15,6 +15,7 @@ export function initMarkupCodeCopy(elMarkup: HTMLElement): void {
     const btn = makeCodeCopyButton();
     // remove final trailing newline introduced during HTML rendering
     btn.setAttribute('data-clipboard-text', el.textContent.replace(/\r?\n$/, ''));
-    el.after(btn);
+    const btnContainer = el.closest('.code-block-container') ?? el.closest('.code-block');
```

Comment:
```suggestion
    const btnContainer = el.closest('.code-block-container, .code-block');
```

I think this is equivalent, but shorter.

Reply:
On the other hand, the way before seems slightly more readable.

Reply:
> I think this is equivalent, but shorter.

No, we only want to use `.code-block-container` if it exists no matter `.code-block` exists or not.

Reply:
```suggestion
    // we only want to use `.code-block-container` if it exists, no matter `.code-block` exists or not.
    const btnContainer = el.closest('.code-block-container') ?? el.closest('.code-block');
```